clients/repo_client.go

```diff
@@ -38,6 +45,8 @@ type RepoClient interface {
 	LocalPath() (string, error)
 	GetFileContent(filename string) ([]byte, error)
 	GetBranch(branch string) (*BranchRef, error)
+	GetCommitSHA() string
+	GetPlatform() string
```

Comment:
this is a breaking change, do we know if allstar is still the only implementer of custom clients? What about legitify?

Reply:
Oh right... what do you think of just a bare `fmt.Printf` in `clients/gitlabrepo/commits.go`? I had that originally but it felt janky so I did this whole thing

Reply:
It's a little hacky, yeah. At the minimum, we need to ensure it's printed to stderr. Printing to stdout breaks scorecard action at the very least